src/event/handlers/LogMessageDeleteHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { Constants, MessageEmbed, Message, TextChannel } from "discord.js";
+import { LOG_CHANNEL_ID } from "../../config.json";
+import EventHandler from "../../abstracts/EventHandler";
+
+class LogMessageUpdateHandler extends EventHandler {
```

Comment:
This should be `LogMessageDeleteHandler`, same for the export.

Reply:
done